internal/proxy/task_upsert.go

```diff
@@ -415,10 +416,18 @@ func (it *upsertTask) deleteExecute(ctx context.Context, msgPack *msgstream.MsgP
 		ts := it.upsertMsg.DeleteMsg.Timestamps[index]
 		_, ok := result[key]
 		if !ok {
+			msgid, err := it.idAllocator.AllocOne()
+			if err != nil {
+				return merr.WrapErrIDAllocatorFailed("Fail to alloc msgid when deleteExecute in upsert")
```

Comment:
The original error lost, just `errors.Wrap(err, "failed to allocate MsgID for delete of upsert")`